misc/release.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+git tag ${tag?no tag specified}
+git push --tags -u origin master
+git checkout stable
+git merge master
+git push -u origin stable
+clear
+python misc/make_changelog.py $tag
```

Comment:
I believe this part should probably in another PR...

Reply:
t4po, I guess you want this for a long time: github.com/actions/create-release, iapr of course.